dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolutionException.java

```diff
@@ -60,6 +66,33 @@ public RepositoryResolutionException(ResolutionException cause, Collection<Strin
         this.featureConflicts = featureConflicts;
     }
 
+    /**
+     * @param object
+     * @param missingTopLevelRequirements
+     * @param missingRequirementNames
+     * @param missingProductInformation        all the product information requirements that could not be found. Can be empty but must not be <code>null</code>
+     * @param allRequirementsResourcesNotFound The {@link MissingRequirement} objects that were not found. Must not be <code>null</code>.
+     * @param featureConflicts                 the details of any feature conflicts which occurred during feature resolution, as returned from {@link Result#getConflicts()}
+     * @param resolvedPlatforms
+     * @param missingPlatforms                 Unknown platform names
+     * @param missingBasePlatforms             unresolved versionless features needing platforms defined
+     */
+    public RepositoryResolutionException(ResolutionException cause, Collection<String> topLevelFeaturesNotResolved, Collection<String> allRequirementsNotFound,
```

Comment:
Copyright date on this class needs to be updated.

Reply:
done